imports/ui/signup/signup.actions.js

```diff
@@ -8,16 +8,15 @@ export function submitSignupInfo (info) {
     Accounts.createUser({
       email: info.emailAddress,
       password: info.password,
+      tac: {
```

Comment:
This is no longer needed as you're setting the contents of "tac" in server side code